src/System.Net.Sockets/ref/4.1.0/System.Net.Sockets.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yeah, I thought so too.  But without this, I get errors when building for netcore50, telling me I haven't implemented the new methods.  It seems like at least the API compat checker is *not* doing the "harvesting" thing, and instead is just using the new contract I just built.